lib/cartopy/mpl/geoaxes.py

```diff
@@ -219,6 +219,20 @@ def inverted(self):
                                         self.source_projection)
 
 
+# A small wrapper that does not create a "proto-Tick", which is expensive and
+# unused with GeoAxes.
+class _TicklessXAxis(matplotlib.axis.XAxis):
+    def reset_ticks(self):
+        del self.majorTicks[:]
```

Comment:
It always looks a little dramatic when `del` is used in module code! From a very small amount of testing it looks like you could instead set the major and minor ticks either to `None` or to `[]`, which would avoid the use of `del`.